README.md

```diff
@@ -73,6 +75,17 @@ or
 docker-compose exec web /bin/sh -c "bundle exec rubocop app config db lib spec Gemfile --format clang"
 ```
 
+### Prettier
+
+Auto-formatting is available using [prettier](https://github.com/prettier/plugin-ruby). To run on the entire project:
+
+```bash
+yarn
+./node_modules/.bin/prettier --write '{app,config,lib,spec}/**/*.rb'
```

Comment:
can we not bundle with `bundle exec rake`

Reply:
I don't think we should; I don't think the build should fail if it fails formatting.

Reply:
cool, so no more linting failed build reason? does that mean we need t change CI/travis/vsts?

Reply:
Nah, I think we should keep the current rubocop setup, but not add prettier to rake at this stage.